parachain/runtime/tests/mock/mod.rs

```diff
@@ -945,6 +946,18 @@ impl ExtBuilder {
             // initialize btc relay
             let _ = TransactionGenerator::new().with_confirmations(7).mine();
 
+            assert_ok!(
+                Call::ExchangeRateOracle(ExchangeRateOracleCall::insert_authorized_oracle(
+                    account_of(ALICE),
+                    vec![]
+                ))
+                .dispatch(root())
+            );
+            assert_ok!(
+                Call::ExchangeRateOracle(ExchangeRateOracleCall::set_btc_tx_fees_per_byte(3, 2, 1))
```

Comment:
Do we want to keep the other fee rates? As discussed in the sync, there may not be a good reason to ever use the low or even high prices.

Reply:
Good question. The other ones are not used at all, so they can safely be removed. The only reason for keeping them around is in case we want to tweak the fees in the future. Which I think is quite likely, because I imagine the users are not going to like the high transaction fees. On the other hand, we can re-add them any time we need them

Reply:
Maybe we can leave them for now, the only concern is increased storage / write cost for the oracle.